masterlist.yaml

```diff
@@ -1050,6 +1050,15 @@ plugins:
   - name: 'RDO - AFT v1.66 Patch.esp'
     after:
       - 'Relationship Dialogue Overhaul.esp'
+  - name: 'Serana Dialogue Edit.esp'
+    url: ['https://www.nexusmods.com/skyrimspecialedition/mods/16222/']
+    after:
+      - 'Book Covers Skyrim.esp'
+      - 'Relationship Dialogue Overhaul.esp'
```

Comment:
Set to load after both as per Mod Author instruction. There is a patch available for book covers but it will won't be needed after the next update, and it's only one texture.